**1) CircRNA circ_POLA2 promotes cervical squamous cell carcinoma progression via regulating miR-326/GNB1**

ORIGINAL RESEARCH ARTICLE

Abstract

Circular RNAs (CircRNAs) are a group of non-coding RNAs that have essential function in the development and progression of various cancers. The expression pattern and function of circRNA in cervical squamous cell carcinoma (CESC) is not fully understood.In the present study, we aimed to investigate the expression profiles and regulation mechanism of circRNA circ_POLA2 in CESC. Circ_POLA2 was highly expressed in CESC tissues and positively correlated with poor prognosis in CESC patients.Knockdown of circ_POLA2 using shRNA inhibited cervical cancer cell proliferation, migration and invasion both in vitro and in vivo. Mechanistically, circ_POLA2 was identified to function as a ceRNA by sponging microRNA-326 (miR-326). Furthermore, miR-326 negatively regulated G protein subunit Beta 1 (GNB1) by targeting its 3'-UTR.Intriguingly, we found that GNB1 was overexpressed and associated with poor prognosis in CESC patients. Overexpression GNB1 could antagonize the inhibitory effect of miR-326 on cervical cancer cell proliferation, migration and invasion. In addition, we demonstrated that circ_POLA2/miR-326/GNB1 axis regulated ERK signaling. In conclusion, circ_POLA2 promotes cervical squamous cell carcinoma development and progression via regulating miR-326/GNB1, which might serve a novel therapeutic target for CESC patients.

Comments:
In general, it is not a good abstract.  
1, Novelty is not good enough, no new concept or new method were created. Just use traditional method to conduct a traditional biology research. In such situation, the solidity will be the most interesting part I want to focus on. However, the author did not provide the details, for example, why they focus on this circRNA?  How many circRNA were detected in this study? Whether multiple test correction were applied in the statistic method? Whether the circRNA expression were validated by northern plot or in situ hybridization? 
The author claimed circ_PLOA2 is sponger of miRNA-326, how they find this miRNA? Did they validated this result, what method was used? Whether the method is solid or not? Same with gene GNB1, whether this method is solid enough to support deep inference?
In addition, why circ_PLOA2 is over-expressed in cancer? The regulatory mechanism is much interesting since current we donâ€™t have good theory to explain the abnormal expression of circRNA. 
When the authors use shRNA to Knockdown circRNA, whether the host gene were influenced? How to make sure the cellular effect they observed is caused by circRNA expression change? Is there any off-target effect? 

**2) Inference of Subpathway Activity Profiles Reveals Metabolism Abnormal Subpathway Regions in Glioblastoma Multiforme**

ORIGINAL RESEARCH ARTICLE

Abstract

Glioblastoma, also known as glioblastoma multiforme (GBM), is the most malignant form of glioma and represents 81% of malignant brain and central nervous system (CNS) tumors. Like most cancers, GBM causes metabolic recombination to promote cell survival, proliferation, and invasion of cancer cells. In this study, we propose a method for constructing the metabolic sub-pathway activity score matrix to accurately identify abnormal targets of GBM metabolism. By integrating gene expression data from different sequencing methods, our method identified 25 metabolic pathways that were significantly abnormal in the GBM patient population, and most of these sub-pathways have been reported to have an effect on GBM. Through the analysis of 25 GBM-related metabolic sub-pathways, we found that (S)-2,3-Epoxysqualene, which was at the central region of the sterol biosynthesis sub-pathway, may have a greater impact on the entire pathway, suggesting a potential high association with GBM. Analysis of CCK8 cell activity indicated that (S)-2,3-Epoxysqualene can inhibit the activity of U87-MG cells in a dose- and time-dependent manner. By flow cytometry, we demonstrated that (S)-2,3-Epoxysqualene not only arrested the U87-MG cell cycle in the G0/G1 phase but also induced cell apoptosis. In summary, we developed a method to identify the metabolic sub-pathways associated with GBM, and we experimentally determined that (S)-2,3-Epoxysqualene had a potential therapeutic effect on GBM.

Comments:

It is a metabolism cancer research, In general, it is a good study, but the abstract is not good enough.  
Pros:  New method was proposed: a new method for constructing the metabolic sub-pathway activity score matrix to accurately identify abnormal targets of GBM metabolism
Cons: More result should be showed in the abstract. For example, (S)-2,3-Epoxysqualene should not be the only one target identified by his new method. The authors should first provide a big picture what they find? And then select few candidate to explain their result was reliable with web-lab experiment validation. The author should also provide some known or positive result we have already know, and their method can identify these known or positive result. 
Meanwhile, as a bioinformatics research, the author should open the analysis pipeline so that others can use his pipeline to repeat his research and apply his pipeline to these own dataset which will be receive better citation rather than just report the performance of his pipeline/method. 

